pkg/services/k8s/admission/admission_control.go

```diff
@@ -63,6 +63,7 @@ type AdmissionError interface {
 type MutatingResponse struct {
 	// PatchOperations is the list of patch ops to perform on the request as part of the mutation
 	PatchOperations []PatchOperation
+	Raw             runtime.Object
```

Comment:
it seems weird to have both `PatchOperations` and `Raw` as options for the response. i commented on the PR that we copied this from: https://github.com/grafana/grafana-app-sdk/pull/92/files#r1142036725

Reply:
Agreed.

Doing a diff and building a patch operation should not be specific to the mutation for a CR so I think we should ONLY respond the mutated object and then use the same mechanisms on all CRD's to create a patch rather than implementing patch logic in every mutation controller.